olm/build-manifests.sh

```diff
@@ -57,22 +56,9 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: stackable-${OPERATOR}
 
-  com.redhat.openshift.versions: v4.12-v4.15
+  com.redhat.openshift.versions: v4.11-v4.17
```

Comment:
is this provisional or are we going to go with 4.17?

Reply:
as discussed. it's just a placeholder that get's overwritten from the command  line